pages/room/[id].js

```diff
@@ -98,7 +105,18 @@ export default function Room(props) {
     }
   }, [userSubscription])
 
+  useEffect(() => {
+    if (roomSubscription.data) {
+      setHidden(!roomSubscription.data.roomUpdated.isShown)
+    }
+  }, [roomSubscription])
+
   if (!pageState && users) {
+    const usersIDList = []
```

Comment:
Rework this to use the room data. users and room might become inconsistent. Also this constantly adds the current user to the end of the users list ie [ 1, 2, 3, 1, 1, 1, 1]